api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -2355,10 +2355,13 @@
 			Add default sortWeights to all current PersonAttributeTypes
 		</comment>
 		<sql>
+		   <![CDATA[CREATE TEMPORARY table temp as(SELECT * from person_attribute_type);
+		    ]]>
 			<![CDATA[
-			UPDATE  person_attribute_type as A JOIN person_attribute_type as B ON A.name = B.name  SET A.sort_weight = (select count(*) from (select * from person_attribute_type) pat where pat.name < A.name) WHERE A.sort_weight is null;
+			UPDATE  person_attribute_type as A JOIN person_attribute_type as B ON A.name = B.name  SET A.sort_weight = (select count(*) from (select * from temp) pat where pat.name < A.name) WHERE A.sort_weight is null;
 			]]>
 		</sql>
+		<dropTable tableName="temp"/>
```

Comment:
@wluyima regular drop worked for me when i wrote <dropTemporaryTable tableName="temp"/>
it gave me error